.github/workflows/openapi-generator.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will regenerate the client library API bindings based on the
+# public OpenAPI spec.
+
+name: OpenAPI Generator
+
+on:
+  workflow_dispatch:
+    inputs: null
+  schedule:
+    - cron: "0 * * * *"
```

Comment:
I know you said the minutes don't count towards anything, but I still think dropping this down to once a day is probably better. Not mandatory change.